src/lineup/internal/cmds.ts

```diff
@@ -184,15 +184,18 @@ export function setGroupCriteria(provider: IObjectRef<any>, rid: number, columns
 export async function setColumnImpl(inputs: IObjectRef<any>[], parameter: any) {
   const p: LocalDataProvider = await resolveImmediately((await inputs[0].v).data);
   const ranking = p.getRankings()[parameter.rid];
-  const prop = parameter.prop[0].toUpperCase() + parameter.prop.slice(1);
+  let prop = parameter.prop[0].toUpperCase() + parameter.prop.slice(1);
+  if (prop === 'RendererType') {
+    prop = 'Renderer';
+  }
 
   let bak = null;
   const waitForSorted = dirtyRankingWaiter(ranking);
   let source: Column | Ranking = ranking;
   if (parameter.path) {
     source = ranking.findByPath(parameter.path);
   }
-  ignoreNext = `${parameter.prop}Changed`;
+  // ignoreNext = `${parameter.prop}Changed`;
```

Comment:
Why does this line needs to be commented out?

Reply:
It is not used anywhere, it is just declared.

Reply:
Edit: Commenting out this variable produces the above bug. It is used outside the function.
It shouldn't therefore be commented out.